.github/workflows/govulncheck.yaml

```diff
@@ -0,0 +1,122 @@
+on:
+  workflow_call:
+    inputs:
+      instance:
+        required: true
+        type: string
+
+name: Govulncheck check
+jobs:
+  govulncheck:
+    runs-on: ${{ inputs.instance }}
+    steps:
+      - name: Checkout the code
+        uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+
+      - name: Install golang
+        run: |
+          ./tests/install_go.sh -f -p
+          echo "/usr/local/go/bin" >> "$GITHUB_PATH"
+
+      - name: Install govulncheck
+        run: |
+          go install golang.org/x/vuln/cmd/govulncheck@latest
+          echo "${HOME}/go/bin" >> "$GITHUB_PATH"
+
+      - name: Build all runtime binaries
+        run: |
+          cd src/runtime
+          make all
+        env:
+          SKIP_GO_VERSION_CHECK: "1"
+
+      - name: Run govulncheck on all runtime binaries
+        run: |
```

Comment:
Let's extract this script into its own file for readability and shellcheck.

Reply:
Good point. Extracted into a separate script instead.